packages/sdk/src/experimental/index.ts

```
@@ -0,0 +1,64 @@
+
+    return {
+        getEntities: async (query: torii.Query) => {
+            return parseEntities(await client.getEntities(query));
+        },
+        getEvents: async (query: torii.Query, historical: boolean = false) => {
+            const events = await client.getEventMessages(query, historical);
+            return historical
+                ? parseHistoricalEvents(events)
+                : parseEntities(events);
+        },
+        subscribeEntities: async (query: torii.Query, callback: Function) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)